frontend/src/components/Login.tsx

```diff
@@ -24,8 +24,13 @@ const Login: React.FC = () => {
         <img src={Logo} alt="logo" className="logo" />
         <h1 className="project-name">TMA</h1>
         <h2 className="moto">Your faithfull task manager</h2>
-        <input type="image" src={Button} onClick={handleGoogleResponseForSignIn} alt=""/>
-        <button onClick={handleGoogleResponseForSignOut}>Google Signout</button>
+        <button className="login-btn" onClick={handleGoogleResponseForSignIn}>
+          <div className="google-logo-container">
```

Comment:
putting <div> or  <p> tags into the button is semantically incorrect

Reply:
![image](https://user-images.githubusercontent.com/56728046/108242442-96bc1700-7155-11eb-952b-b2a501197468.png)